File: python/count_pos_sum_neg.py
##https://www.codewars.com/kata/576bb71bbbcf0951d5000044
def count_positives_sum_negatives(arr):
    min = 0
    max = 0
    res = []
    for x in arr:
        if x > 0:
            max += 1
        if x < 0:
            min += x
    if len(arr) > 0:
        res.append(max)
        res.append(min)
    return res

File: rust/count_pos_sum_neg.rs
//https://www.codewars.com/kata/576bb71bbbcf0951d5000044
fn count_positives_sum_negatives(input: Vec<i32>) -> Vec<i32> {
    let mut res:Vec<i32> = Vec::new();
    let mut pos:i32 = 0;
    let mut neg:i32 = 0;
    for &num in &input {
        if num > 0 {
            pos += 1;
        }
        else if num < 0 {
            neg += num;
        }
    }
    if input.len() > 0 {
        res.push(pos);
        res.push(neg);
    }
    res
}